public/app/features/explore/Explore.tsx

```diff
@@ -183,10 +184,40 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
     }
   };
 
+  /**
+   * Used by Logs details.
+   * Returns true if all queries have the filter, otherwise false.
```

Comment:
```suggestion
   * Returns true if all queries have the filter, otherwise false.
   TODO: In the future, we would like to return active filters based the query that produced the log line
```

Reply:
Updated!